FabiWare/commands.cpp

```diff
@@ -28,13 +28,14 @@ const struct atCommandType atCommands[] PROGMEM = {
     {"TS"  , PARTYPE_UINT },  {"TP"  , PARTYPE_UINT }, {"MA"  , PARTYPE_STRING},{"WA"  , PARTYPE_UINT  },
     {"TT"  , PARTYPE_UINT },  {"AP"  , PARTYPE_UINT }, {"AR"  , PARTYPE_UINT},  {"AI"  , PARTYPE_UINT  },
     {"FR"  , PARTYPE_NONE },  {"BT"  , PARTYPE_UINT }, {"BC"  , PARTYPE_STRING},{"DP" , PARTYPE_UINT  },
-    {"AD"  , PARTYPE_UINT  } 
+    {"AD"  , PARTYPE_UINT },  {"SC"  , PARTYPE_STRING }
 };
 
 void printCurrentSlot()
 {
         Serial.print(F("Slot:"));  Serial.println(settings.slotname);
         Serial.print(F("AT WS ")); Serial.println(settings.ws);
+        Serial.print(F("AT SC ")); Serial.println(settings.sc);
```

Comment:
this is problematic because the data format for sending values should be the same as for receiving ("0xaabbcc")
I will add a conversion function